src/LLVM_intrinsics.jl

```diff
@@ -802,6 +802,36 @@ end
     )
 end
 
+###########
+# Bitmask #
+###########
+
+@generated function bitmask(x::LVec{N, Bool}) where {N}
+    if N > 128
+        return :(throw(ArgumentError(("vector length $(N) must be <= 128"))))
```

Comment:
Where does this restriction come from? Can you add a comment explaining this?

Reply:
The main reason is that UInt128 is the largest unsigned integer Julia exports. Otherwise, I get the following error:
```
UndefVarError: `UInt256` not defined in `Main`

In theory I can remove this restriction. If someone wanted to they can just import UInt256 (or larger types) from BitIntegers.

Reply:
The restriction itself is fine – just add a comment with the explanation.

Reply:
After looking at it again, I kind of like the option of making it work with UInt256 by import from BitIntegers. However, I also understand that `UndefVarError` can be confusing for the unsuspecting user.

Would you like the restriction to be added back or are you fine with letting the user import larger types if they need them?

Reply:
Feels weird to me, in that case could we use our own `primitive type` (like `BitPattern` or something) that is exactly the size of the number of bits needed?

Reply:
Ok, seems like the easiest path forward (for now at least) is to re-add the restriction with a comment.